### ISSUE TITLE
/usr/bin/env: ‘node\r’: No such file or directory

### ISSUE THREAD

**Issue by mpalourdio** [gios/gzipper#5]
Hello,

After bumping from 1.4.2 to 1.5.1, compression fails (on linux) with this message. Same error with latest v2.x release.

```bash
$ gzipper ./dist
/usr/bin/env: ‘node\r’: No such file or directory  
error Command failed with exit code 127.  
```

node version : 10.15.1

Sticking to 1.4.2 fixes the problem.

Thanks for investigating !

**Comment by mpalourdio** [comment]
Quick googling indicates that it could possibily be related to CRLF/LF issue with git.

**Comment by gios** [comment]
@mpalourdio Hi, I've published a new version **2.1.0** with LF line ending. Could you try to test it on Linux and give feedback?

**Comment by mpalourdio** [comment]
Awesome, it fixed the problem. Many thanks for this quick release. I use you package quite everyday, it makes my life easier and does the job very well !
